EFCore/OrderRepository.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+
+namespace EFCore
+{
+    public class OrderRepository
+    {
+        private readonly Module14Context _context;
+
+        public OrderRepository(Module14Context context)
+        {
+            _context = context;
+        }
+
+        public async Task<List<Order>> GetAllAsync(int month = 0, int year = 0, OrderStatus orderStatus = 0, int productId = 0)
+        {
+            var orders = await _context.Orders.FromSqlRaw("GetOrders @Month = {0}, @Year = {1}, @OrderStatus = {2}, @ProductId = {3}", month, year, orderStatus, productId).ToListAsync();
```

Comment:
Not a good practice to use SQL with EF directly. Even if we use we should have used Parameterization, otherwise your could is vulnerable for SQL injection